tests/integration/test_vulnerability_detector/test_scan_results/test_scan_nvd_feeds.py

```diff
@@ -6,14 +6,14 @@
 import pytest
 import json
 from time import sleep
+from shutil import copy
+import gzip
```

Comment:
Unused module

Reply:
Done in d7c8e82, also removed unused files.